### DISCUSSION TITLE
Add a new search matching pattern to the create node window

### DISCUSSION THREAD

**Discussion by VintageGameBoy** [godotengine/godot-proposals#11334] [Editor]
Add a new search matching pattern to the create node window
Now you can search for node descriptions
It should be very helpful for non-native English speakers
pull97767 https://github.com/godotengine/godot/pull/97767

![11C53E1C8FA05EFD0BD6E805B8A16430](https://github.com/user-attachments/assets/3dc9571c-84b7-4b0a-9b32-49acf113fef3)
![373257884-b2107f92-b6ce-487f-bbdf-63e7cb395f20](https://github.com/user-attachments/assets/7d83653c-38ad-487a-b16a-5ad7f28efc57)

**Comment by VintageGameBoy** [comment]
I think this feature needs a more appropriate name

**Comment by theraot** [comment]
If I understand correctly, this is to allow the search to look into the description (which is a feature that I would like to have). Correct? Addendum. I'd say put something along those lines in the title.

**Reply by Calinou** [reply]
This sounds interesting, but I'm worried it would be really slow to search, especially if it has to look into the documentation of script-provided classes.

**Reply by VintageGameBoy** [reply]
It runs very fast on my computer
